data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -44,6 +44,9 @@ spec:
             required:
             - installationDisk
             type: object
+          cgroupsV2:
```

Comment:
I think to start we can omit this from the install config and just leave it as something where if you want to turn it on you provide the additional manifests.  The installer doesn't need to be involved IMO.

(The ugly exception here is the bootstrap node, but hopefully it works as v1 to start)

See also https://github.com/openshift/ci-chat-bot/issues/112 where I argue that we should have a documented set of example "day 1 manifests" in a git repo like github.com/openshift/ci-cluster-variants or something.

Reply:
TL;DR I don't think we need this patch - *except* if the bootstrap node is relevant, and hopefully it isn't.

(I'm not arguing strongly against this either...perhaps instead this manifests as `env OPENSHIFT_INSTALL_CGROUPS_V2=1` or something for convenience.  But it's more of a developer convenience instead of something we should put in front of customers I think until we enable it by default.  (Right?))